packages/vue/src/components/styles.css

```diff
@@ -14,10 +14,6 @@
   @apply flex flex-row items-center space-x-1;
 }
 
-[data-amplify-wrapper] [data-amplify-fieldset] {
-  @apply grid gap-6;
-}
-
```

Comment:
Should we remove all of these `styles.css` and entirely defer to `@aws-amplify/ui/styles.css`?

Reply:
I don't think so. It would break the look and feel of the Vue app. We have these styles in Angular and React too. 

Maybe eventually we'll need to get rid of these...